Automattic/wp-calypso
# Reader: add Customer Council banner

Closes https://github.com/Automattic/dotcom-forge/issues/9343

## Proposed Changes

* Add new banner to drive subscriptions to customer council P2

### Mobile

<img width="376" alt="image" src="https://github.com/user-attachments/assets/c7120770-4fdb-462c-b406-033c39f3c1ab">

### Desktop

<img width="992" alt="image" src="https://github.com/user-attachments/assets/0fd6c8fa-601a-4515-b4c0-f31db19ee872">

## Why are these changes being made?


* To gather well-rounded feedback, we’ll show a dismissible banner in the Reader’s following feed for 24 hours to start.

## Testing Instructions



* Navigate to `/read` or `/discover`
* Click "subscribe" on banner
* You should navigate to (https://readercouncilgeneral.wordpress.com/)
* You should be subscribed to the P2
* Navigate back to a Reader feed and the banner should not appear
* Repeat on mobile

## Pre-merge Checklist
